cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -0,0 +1,172 @@
+import os
+
+import yaml
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_imagebuilder as imagebuilder
+from aws_cdk import aws_ssm as ssm
+from aws_cdk import core
+
+from common.utils import load_yaml
+from pcluster.imagebuilder import ImageBuilder
+
+
+class ImageBuilderStack(core.Stack):
+    """Create the Stack for imagebuilder."""
+
+    def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBuilder, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        self.imagebuild = imagebuild
+        image_config = imagebuild.config.image_config
```

Comment:
By looking at this code it seems to me that we should review the design of this part.
I don't like accessing to config then image_config, we should instead use something like:

imagebuild.image.root_volume.size directly without the need to navigate the config items.
Feel free to go ahead but I think we'll refactor this part.

Reply:
Yes, as far as I know, we will refactor the config and core layer to ImageBuilder and ImageBuilderManager, I just keep the logic now and will modify it after the new design of this part is released